android/java/org/chromium/chrome/browser/rewards/BraveRewardsPanel.java

```diff
@@ -1144,11 +1118,14 @@ && getDaysUntilRewardsPayment(nextPaymentDate) > 0) {
 
     @Override
     public void onGetUserType(int userType) {
-        if (UserType.LEGACY_UNCONNECTED == userType
-                && mBraveRewardsNativeWorker.getVbatDeadline() > System.currentTimeMillis()) {
-            showVbatExpireNotice();
-        } else if (UserType.UNCONNECTED == userType) {
-            newInstallViewChanges();
+        if (mExternalWallet == null) {
```

Comment:
If the user is unconnected, then `mExternalWallet` will be null. Do we want to skip the code below in that case?